Src/UIComponents/utils/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const createElement = (type: string, className: string = '') => {
+    const createdElement = document.createElement(type);
+    className && (createdElement.className = className);
```

Comment:
Ciekawe zastosowanie operatora && ;)
Inną opcją jest if(className) createdElement.....;
...Ale myślę że może to zostać jak jest.